website/content/en/releases/13.4R/relnotes.adoc

```diff
@@ -21,7 +21,7 @@ include::shared/en/urls.adoc[]
 
 [.abstract-title]
 The release notes for FreeBSD {releaseCurrent} contain a summary of the changes made to the FreeBSD base system on the {releaseBranch} development line.
-This document lists applicable security advisories that were issued since the last release, as well as significant changes to the FreeBSD kernel and userland.
```

Comment:
No mention of "FreeBSD-SA" or "CVE" in the document … am I missing something?

Either add the advisories to this document, or (as I have done) remove the part of the sentence that seems to describe things that are not present.

Reply:
SAs present, thanks: 

<https://www.freebsd.org/releases/13.4R/relnotes/#security>